yamdwe.py

```diff
@@ -49,8 +49,9 @@ def main():
     pages = importer.get_all_pages()
     print("Found %d pages to export..." % len(pages))
 
-    # Add a shameless "exported by yamdwe" note to the front page of the wiki
+    # Add a shameless "exported by yamdwe" note to the front page of the wiki - really shameless, but I'll keep it
```

Comment:
If the note really bugs you then feel free to send a new PR that removes the "exported by yamde" note entirely. I'd be OK with accepting it. (I always expect people will delete the note after they export anyhow.)

But could you update the PR to remove this comment and the whitespace change below, please?